[i18n-KO] Translated index page to Korean

wonhyeongseo opened the pull request:
# What does this PR do?

Translated the `Getting started - index.mdx` file of the documentation to Korean. I use transformers for my side-project and wanted to contribute. As a beginner, this seemed appropriate.

Some questions:
- I can't access the pr documentation endpoint for Korean. Is there a missing step?
- I left all the other files alone after copying the English docs. Is this acceptable for future updates or should I add contents to the `_toctree.yml` only when translation is complete? Should I delete all other files for a cleaner review?
- In Korea, people use both `Transformer` and `트랜스포머` to describe the model. However, as this is more of a product of HuggingFace I opted not to translate it. May you please let me know which you would prefer?

Thank you in advance for your review. 

<!-- Remove if not applicable -->

Part of https://github.com/huggingface/transformers/issues/20179


## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?


## Who can review?
@sgugger, may you please review this PR?

<!-- Your PR will be replied to more quickly if you can figure out the right person to tag with @

 If you know how to use git blame, that is the easiest way, otherwise, here is a rough guide of **who to tag**.
 Please tag fewer than 3 people.

Models:

- albert, bert, xlm: @LysandreJik
- blenderbot, bart, marian, pegasus, encoderdecoder,  t5: @patrickvonplaten, @patil-suraj
- longformer, reformer, transfoxl, xlnet: @patrickvonplaten
- fsmt: @stas00
- funnel: @sgugger
- gpt2: @patrickvonplaten, @LysandreJik
- rag: @patrickvonplaten, @lhoestq
- tensorflow: @LysandreJik

Library:

- benchmarks: @patrickvonplaten
- deepspeed: @stas00
- ray/raytune: @richardliaw, @amogkam
- text generation: @patrickvonplaten
- tokenizers: @n1t0, @LysandreJik
- trainer: @sgugger
- pipelines: @LysandreJik

Documentation: @sgugger

HF projects:

- datasets: [different repo](https://github.com/huggingface/datasets)
- rust tokenizers: [different repo](https://github.com/huggingface/tokenizers)

Examples:

- maintained examples (not research project or legacy): @sgugger, @patil-suraj
- research_projects/bert-loses-patience: @JetRunner
- research_projects/distillation: @VictorSanh

 -->

wonhyeongseo committed:
docs: translated index page to korean

docs: i18n: first draft of index page
docs: fix: first revision of index page
docs: i18n: missed section - supported frameworks
docs: fix: second revision of index page
review by @ArthurZucker

refactor: remove untranslated files from korean
docs: fix: remove untranslated references from toctree.yml
feat: enable korean docs in gh actions
docs: feat: add in_translation page as placeholder
docs: bug: testing if internal toc need alphabet chars
docs: fix: custom english anchor for non-alphanumeric headings
review by @sgugger

docs: i18n: translate comments on install methods in _config.py
docs: refactor: more concise wording for translations

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_20180). All of your documentation changes will be reflected on that endpoint.

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_20180). All of your documentation changes will be reflected on that endpoint.

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_20180). All of your documentation changes will be reflected on that endpoint.

ArthurZucker commented:
Hey! 만나서 반가워요! Would love to hear more about the project you were working on and how transformers is used in Korea! 🤗 

Thanks a lot for your work! 
- The PR documentation endpoint was updating and should now be visible! The `KO` is not appearing because I think you did not add the file to the `toctree` indeed.
- You should delete all the other files! We don't keep template files and if someone wants to translate for futur updates, they will just copy the single file 😄 
-  I think you did a great job in keeping `Transformer` as it is indeed more the name of the library! When talking about actual `transformers` architecture, its up to you, and depends on what is more used/understandable! 

I will be glad to review once you have cleaned up the modified files! 👍🏻 Also cc @eunseojo just FYI 😉

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_20180). All of your documentation changes will be reflected on that endpoint.

wonhyeongseo commented:
Hello @ArthurZucker, 저도 만나서 반가워요! Thank you so much for your warm welcome and guidance.
Nice to meet you, @eunseojo 🤗 

- I added relevant updates to `_toctree.yml` as you suggested and also added a language code to github worflows.
- So much simpler now! I translated the big section titles as they were mentioned in the `index.mdx` page.
- Yay 😆 thank you very much for your compliment hehe

p.s. I'm just a beginner but Korea, at least a community I learn lots from- [PseudoLab](https://pseudo-lab.com/), is extensively using huggingface. My side-project goal is to enhance "google translator-ish" results: **to a more friendly, neighbourhood tone** in Korean (_grammar checking included_!) I'm still tinkering, just learning mostly how to get this to work. Thank you for your interest!

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_20180). All of your documentation changes will be reflected on that endpoint.

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_20180). All of your documentation changes will be reflected on that endpoint.

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_20180). All of your documentation changes will be reflected on that endpoint.

sgugger commented on docs/source/ko/_config.py:
```diff
@@ -0,0 +1,14 @@
+# docstyle-ignore
+INSTALL_CONTENT = """
+# Transformers installation
+! pip install transformers datasets
+# To install from source instead of the last release, comment the command above and uncomment the following one.
```
Would be nice to translate the two comments in Korean!

sgugger reviewed (approved):
Looking great, thanks a lot for working on this!

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/transformers/pr_20180). All of your documentation changes will be reflected on that endpoint.

sgugger commented:
Thanks for working on this! Excited to see a new doc in Korean!
